common/src/main/java/org/wso2/testgrid/common/TestScenario.java

```diff
@@ -75,6 +78,14 @@
     @Element(description = "holds the test engine type (i.e. JMETER, TESTNG)")
     private TestEngine testEngine;
 
+    @Column(name = "pre_script")
```

Comment:
Since this is a boolean holding the status, shall we name the database column as `is_pre_script_success`?